apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -645,22 +645,21 @@ wm_parse_commands(struct options *opt, int argc, char *argv[])
 		opt->func = wm_scan;
 	} else if (strcmp(argv[2], "auto") == 0) {
 		opt->func = wm_auto_test;
-		if (argc != 7) {
+		if (argc < 7) {
```

Comment:
I think if (argc != 7) is better.

Reply:
the number of parameters is determined by security type. so it can be 7 or 8 but not under 7.